.storybook/webpack.config.js

```diff
@@ -7,7 +7,19 @@ const welcomeSectionPath = path.resolve(__dirname, './');
 const utilsPath = path.resolve(__dirname, '../utils');
 const postcssConfigPath = path.resolve(__dirname, './postcss.config');
 
-module.exports = ({config, mode}) => {
+module.exports = ({ config, mode }) => {
+  // This is so we get consistent results when loading .ts/tsx and .mdx files
+  const babelPresetEnvConfig = [
```

Comment:
We use this in a couple places in the config (.ts/.tsx and .mdx files) so I stored it here.